stdlib/public/runtime/Backtrace.cpp

```diff
@@ -451,17 +474,6 @@ BacktraceInitializer::BacktraceInitializer() {
   }
 
   if (_swift_backtraceSettings.enabled == OnOffTty::On) {
-#if TARGET_OS_OSX || TARGET_OS_MACCATALYST
-    // Make sure that all fds are closed except for stdin/stdout/stderr.
```

Comment:
Seems like this would still be nice (albeit not very important), but I assume it's not worth the cost of keeping `backtraceSpawnAttrs` around forever just in case a crash occurs?

Reply:
Yes, the issue there is that the attribute storage allocates a fair sized chunk of memory on Darwin and so we wouldn't want this turned on more widely as it stands. But as long as we're checking up front for get-task-allow, it doesn't matter so much anyway — the program has already been marked as suitable for debugging, so someone with a debugger could already get access to the file descriptors.